rl_coach/presets/RoboSuite_CubeExp_TD3_Intrinsic_Reward.py

```diff
@@ -0,0 +1,102 @@
+from rl_coach.agents.td3_exp_agent import TD3IntrinsicRewardAgentParameters
+from rl_coach.architectures.embedder_parameters import InputEmbedderParameters
+from rl_coach.architectures.layers import Dense, Conv2d, BatchnormActivationDropout, Flatten
+from rl_coach.base_parameters import EmbedderScheme
+from rl_coach.core_types import TrainingSteps, EnvironmentEpisodes, EnvironmentSteps
+from rl_coach.environments.robosuite_environment import RobosuiteGoalBasedExpEnvironmentParameters, \
+    OptionalObservations
+from rl_coach.filters.filter import NoInputFilter, NoOutputFilter
+from rl_coach.graph_managers.basic_rl_graph_manager import BasicRLGraphManager
+from rl_coach.graph_managers.graph_manager import ScheduleParameters
+from rl_coach.architectures.head_parameters import RNDHeadParameters
+from rl_coach.schedules import LinearSchedule
+
+####################
+# Graph Scheduling #
+####################
+
+schedule_params = ScheduleParameters()
+schedule_params.improve_steps = TrainingSteps(300000)
+schedule_params.steps_between_evaluation_periods = TrainingSteps(300000)
+schedule_params.evaluation_steps = EnvironmentEpisodes(0)
+schedule_params.heatup_steps = EnvironmentSteps(1000)
+
+#########
+# Agent #
+#########
+
+agent_params = TD3IntrinsicRewardAgentParameters()
+agent_params.algorithm.use_non_zero_discount_for_terminal_states = True
+agent_params.exploration.noise_schedule = LinearSchedule(1.5, 0.5, 300000)
+
+agent_params.input_filter = NoInputFilter()
+agent_params.output_filter = NoOutputFilter()
+
+# Camera observation pre-processing network scheme
+camera_obs_scheme = [
+    Conv2d(32, 8, 4),
+    BatchnormActivationDropout(activation_function='relu'),
+    Conv2d(64, 4, 2),
+    BatchnormActivationDropout(activation_function='relu'),
+    Conv2d(64, 3, 1),
+    BatchnormActivationDropout(activation_function='relu'),
+    Flatten(),
+    Dense(256),
+    BatchnormActivationDropout(activation_function='relu')
+]
+
+obs_name = 'camera'
```

Comment:
Use the attribute you added for this:
```suggestion
obs_name = agent_params.algorithm.img_obs_key
```